labcontrol/db/process.py

```diff
@@ -3819,3 +3826,407 @@ def _generate_amplicon_prep_information(self):
+                LEFT JOIN labman.plate sampleplate ON (
+                    samplewell.plate_id = sampleplate.plate_id)
+                LEFT JOIN labman.process platingprpr ON (
+                    samplecpcp.upstream_process_id = platingprpr.process_id)
+                LEFT JOIN labman.primer_composition primercp ON (
+                    libprepcp.i5_primer_composition_id =
+                    primercp.primer_composition_id)
+                LEFT JOIN labman.primer_composition primercp2 ON (
+                    libprepcp.i7_primer_composition_id =
+                    primercp2.primer_composition_id)
+                LEFT JOIN labman.composition primercpcp ON (
+                    primercp.composition_id = primercpcp.composition_id)
+                LEFT JOIN labman.composition primercpcp2 ON (
+                    primercp2.composition_id = primercpcp2.composition_id)
+                LEFT JOIN labman.process primerworkingplateprpr ON (
+                    primercpcp.upstream_process_id =
+                    primerworkingplateprpr.process_id)
+                LEFT JOIN labman.process primerworkingplateprpr2 ON (
+                    primercpcp2.upstream_process_id =
+                    primerworkingplateprpr2.process_id)
+                LEFT JOIN labman.primer_set_composition primersetcp ON (
+                    primercp.primer_set_composition_id =
+                    primersetcp.primer_set_composition_id)
+                LEFT JOIN labman.primer_set_composition primersetcp2 ON (
+                    primercp2.primer_set_composition_id =
+                    primersetcp2.primer_set_composition_id)
+                LEFT JOIN labman.composition primersetcpcp ON (
+                    primersetcp.composition_id = primersetcpcp.composition_id)
+                LEFT JOIN labman.composition primersetcpcp2 ON (
+                    primersetcp2.composition_id =
+                    primersetcpcp2.composition_id)
+                LEFT JOIN labman.well primersetwell ON (
+                    primersetcpcp.container_id = primersetwell.container_id)
+                LEFT JOIN labman.well primersetwell2 ON (
+                    primersetcpcp2.container_id = primersetwell2.container_id)
+                LEFT JOIN labman.plate primersetplate ON (
+                    primersetwell.plate_id = primersetplate.plate_id)
+                LEFT JOIN labman.plate primersetplate2 ON (
+                    primersetwell2.plate_id = primersetplate2.plate_id)
+                FULL JOIN qiita.study_sample USING (sample_id)
+                LEFT JOIN qiita.study as study USING (study_id)
+                WHERE libprepplate.plate_id IN (
+                    SELECT distinct libprepplate2.plate_id
+                    FROM labman.sequencing_process sp
+                    LEFT JOIN labman.sequencing_process_lanes spl USING (
+                        sequencing_process_id)
+                    LEFT JOIN labman.pool_composition_components pcc1 ON (
```

Comment:
In this query, unlike the one it was based on, the pool_composition_components table is only used once.  Therefore, please alias this as `pcc` rather than `pcc1` as we don't have to distinguish it from a second reference to the same table :)